package.json

```diff
@@ -2,8 +2,8 @@
   "name": "@rudderstack/rudder-sdk-node",
   "version": "2.0.3",
   "description": "Rudder Node SDK",
-  "main": "./cjs/index.js",
-  "module": "./esm/index.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
```

Comment:
Why this was changed? This should be the path inside the dist folder once the package.json is in the package

Reply:
Adressed